include/boost/date_time/time.hpp

```diff
@@ -164,7 +164,7 @@ namespace date_time {
     }
     time_type operator+=(const time_duration_type& td)
     {
-      time_ = (time_system::get_time_rep(date(), time_of_day() + td));
+      time_ = time_system::add_time_duration(time_,td);
```

Comment:
I realize we've already pulled this, but I'm just wondering if we really have coverage for split time systems that use things other than single integers here. See this page for info on the 96 bit representations.  My guess is that the options is a) rarely used, b) not unit tested, and c) probably ok -- but we should double check...

https://www.boost.org/doc/libs/1_69_0/doc/html/date_time/details.html#date_time.buildinfo

Reply:
This change wasn't merged yet.